docs/advanced-features/using-mdx.md

```diff
@@ -56,6 +56,27 @@ The following steps outline how to setup `@next/mdx` in your Next.js project:
    })
    ```
 
+   > Using MDX plugins will require using next.config.mjs because all the plugins are [ECMAScript modules](https://nodejs.org/api/esm.html)
```

Comment:
I don't think this is accurate. Could someone confirm?

Reply:
e.g. works fine here https://github.com/leerob/leerob.io/blob/main/next.config.js

Reply:
> I don't think this is accurate. Could someone confirm?

Most plugins are esm modules

Reply:
cc @timneutkens, who told me it was correct.